client/gutenberg/editor/main.jsx

```diff
@@ -40,6 +37,15 @@ class GutenbergEditor extends Component {
 	render() {
 		const { postType, siteId, siteSlug, post } = this.props;
 
+		//see also https://github.com/WordPress/gutenberg/blob/45bc8e4991d408bca8e87cba868e0872f742230b/lib/client-assets.php#L1451
+		const editorSettings = {
+			autosaveInterval: 3, //interval to debounce autosaving events, in seconds.
+			titlePlaceholder: translate( 'Add title' ),
+			bodyPlaceholder: translate( 'Write your story' ),
+		};
+
+		const isAutoDraft = () => post && 'auto-draft' === post.status;
```

Comment:
Minor note, but we can also move this logic to mapStateToProps:

```jsx
const mapStateToProps = ( state, { siteId, postId, uniqueDraftKey } ) => {
	const draftPostId = get( getHttpData( uniqueDraftKey ), 'data.ID', null );
	const post = getPost( siteId, postId || draftPostId );
    const isAutodraft = get( post, 'status', null );
    const overridePost = isAutodraft ? { title: '' } : null;

	return {
		siteSlug: getSiteSlug( state, siteId ),
		post,
        overridePost,
	};
};
```

Reply:
Wow, I really was looking at this too much 🙃 Yes, thanks!